pepper-apis/studybuilder-cli/studies/lms/patches/consent_assent-pdf-v2.conf

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
just copied from existing pediatric pdf config, nothing new. 
copied consent_assent pdf config since other file contained both assent and parental pdf config.